pkgs/development/python-modules/rstcheck/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, docutils }:
+
+buildPythonPackage rec {
+  pname = "rstcheck";
+  version = "3.3.1";
+
```

```suggestion

  disabled = isPy27;

```

and add isPy27 to inputs.

---

Thanks, i didnt know this.